yarn.lock

```diff
@@ -21300,7 +21300,7 @@ slide@^1.1.6:
   resolved "https://registry.yarnpkg.com/slide/-/slide-1.1.6.tgz#56eb027d65b4d2dce6cb2e2d32c4d4afc9e1d707"
   integrity sha1-VusCfWW00tzmyy4tMsTUr8nh1wc=
 
-smart-buffer@^4.0.2, smart-buffer@^4.1.0:
+smart-buffer@^4.0.2, smart-buffer@^4.1.0, smart-buffer@^4.2.0:
```

Comment:
failing on gitlab CI
```
check-lockfile.sh: yarn.lock was modified unexpectedly - terminating
```

Reply:
Yes I can see that is still failing in ci, I a still investigating since that `ci/scripts/check_lockfile.sh` script runs successfully in my computer.

Reply:
you should update lockfile using `yarn` and commit it to fix this